config/i18n/locales/english/translations.json

```
@@ -86,7 +86,8 @@
       "coc": "Code of Conduct",
       "privacy": "Privacy Policy",
       "tos": "Terms of Service",
-      "copyright": "Copyright Policy"
+      "copyright": "Copyright Policy",
+      "powered-by": "Powered by Hashnode"
```

It should be fine to revert this and add the text to the element itself:

```suggestion
      "copyright": "Copyright Policy"
```